composer.json

```diff
@@ -24,6 +28,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.3",
-        "vimeo/psalm": "4.13.1"
+        "vimeo/psalm": "4.13.1",
+        "laravel/pint": "dev-main"
```

Comment:
Let's use the latest version instead of using the main branch
https://github.com/laravel/pint/releases/tag/v1.2.0